payment/src/main/java/com/kyriba/conference/payment/domain/vo/Amount.java

```diff
@@ -0,0 +1,19 @@
+package com.kyriba.conference.payment.domain.vo;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.util.Currency;
+
+/**
+ * @author Igor Lizura
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class Amount {
```

Comment:
Semi-empty DTO.

Reply:
what does it mean? in which way, I have to keep the price?